scenes/world_map/frays_end.tscn

```diff
@@ -134,37 +134,86 @@ editor_draw_limits = true
 [node name="Props" type="Node2D" parent="."]
 y_sort_enabled = true
 
-[node name="NdCrochenthemum01" type="Sprite2D" parent="Props"]
-position = Vector2(1455, 1819)
-texture = ExtResource("40_06x6x")
+[node name="Flowerbed" type="Area2D" parent="Props"]
+y_sort_enabled = true
+position = Vector2(1358.05, 1693.82)
+
+[node name="CollisionPolygon2D" type="CollisionPolygon2D" parent="Props/Flowerbed"]
+polygon = PackedVector2Array(80.9502, -52.8169, 81.9502, 73.1831, 145.95, 74.1831, 146.95, 134.183, 476.95, 136.183, 480.95, 204.183, -87.05, 210.18, -91.05, -178.82, 10.95, -176.82, 11.649, -54.5072)
+disabled = true
+
+[node name="AreaFiller" type="Node" parent="Props/Flowerbed"]
+script = ExtResource("38_cjmkx")
+scenes = Array[PackedScene]([ExtResource("45_cjmkx")])
+metadata/_custom_type_script = "uid://bdhjixygupit1"
+
+[node name="Flower" type="Node2D" parent="Props/Flowerbed" instance=ExtResource("45_cjmkx")]
+position = Vector2(145.987, 76.4082)
+script = ExtResource("9_lftgk")
```

Comment:
This is now pretty satisfactory but I can't understand why the script is being set explicitly on each Flower when it is also set on the root of the instanced scene...